fe/fe-core/src/main/java/com/starrocks/persist/PartitionPersistInfo.java

```diff
@@ -128,6 +198,20 @@ public void readFields(DataInput in) throws IOException {
         if (GlobalStateMgr.getCurrentStateJournalVersion() >= FeMetaVersion.VERSION_74) {
             isTempPartition = in.readBoolean();
         }
+
+        if (GlobalStateMgr.getCurrentStateJournalVersion() >= FeMetaVersion.VERSION_93) {
```

Comment:
Do not increase meta version, otherwise StarRocks version roll back is impossible.
Maybe you can add a new class using json serialization.